PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/AzureIntegerInput.java

```diff
@@ -24,11 +24,19 @@
     @Setter
     private Integer maxValue;
 
+    public AzureIntegerInput() {
+        super();
+        this.setDefaultValue(0);
```

Comment:
We may also need to check whether default value fit min/max limitation

Reply:
user can/should set default value mannually if min/max are set.